.github/workflows/check-linked-issues.yml

```diff
@@ -1,12 +1,16 @@
 name: Check linked issues
 
 on:
-  pull_request_target:
+  pull_request:
```

Comment:
why this change? iirc the reason why we recommend using this this way is because it limits what the workflow can do. pull request target works in the context of the base, rather than the source code contained in the PR branch, meaning that it limits potential malicious actions